brianchandotcom/liferay-portal
# LPS-195493 Convert old AUI user-associated-data-web using JS

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-195493)

## Proposed solution

Convert old AUI user-associated-data-web using JS

## Test Scenario 

1. Go to Control Panel > Users and Organisations.
2. Create a new User and add it Liferay as a site in Membership and Site Administrator in site Roles in Roles. 
3. Save and impersonate the user.
4. Create a Web content. 
5. Go back to Users and organisations and select “Export Personal Data” in the Kebab menu.
6. Add a new 	Export Process selecting the web content created in step 4.
7. Check the behaviour.


https://github.com/liferay-echo/liferay-portal/assets/93203423/86e5e106-04c3-426a-b80b-45fbbda49b36

cc/ @BarbaraCabrera
